docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -108,21 +108,25 @@ Every encrypted Internet connection has an unencrypted part. This is the very fi
 
 #### OCSP checking
 
-Once enabled, this option runs asynchronous OCSP checks to check whether the website’s SSL certificate is revoked.
+This option runs asynchronous OCSP checks to check whether the website’s SSL certificate is revoked.
 
-If the OCSP check is completed within the minimum timeout, AdGuard will immediately block the connection if the certificate is revoked or establish the connection if the certificate is valid.
+If the OCSP check is completed within the minimum timeout, AdGuard immediately blocks the connection if the certificate is revoked or establishes the connection if the certificate is valid.
 
-If the verification takes too long, AdGuard will establish a connection and continue checking the certificate in the background. If it is revoked, current and future connections to the domain will be blocked.
+If the verification takes too long, AdGuard establishes a connection and continues checking the certificate in the background. If it is revoked, current and future connections to the domain are blocked.
```

Comment:
If the verification takes too long, AdGuard allows the connection while continuing to check the certificate status in the background. If the certificate is revoked, current and future connections to the domain are blocked.